src/foam/nanos/crunch/document/Document.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.document',
+  name: 'Document',
```

Comment:
Are we moving crunch Document from nanopay to FOAM? if so, will need to update references on nanopay side and would also need migration.

Reply:
It was suggested by Joel. We didn't find another way to make code less brittle.

Reply:
I'm not sure how we can migrate the data.  I believe we'll have to keep both so we can load and copy.